v2/cmd/mpi-operator/app/server.go

```diff
@@ -157,6 +157,7 @@ func Run(opt *options.ServerOption) error {
 			kubeInformerFactory.Core().V1().ConfigMaps(),
 			kubeInformerFactory.Core().V1().Secrets(),
 			kubeInformerFactory.Core().V1().Services(),
+			kubeInformerFactory.Batch().V1().Jobs(),
```

Comment:
Is this related to current PR?

Reply:
Job is used for launchers, thus we have to watch it.

Reply:
See #391